cronback-lib/Cargo.toml

```diff
@@ -65,3 +65,5 @@ sea-orm-migration = { version = "0.11.0", features = [
   "sqlx-sqlite",              # `DATABASE_DRIVER` feature
   "sqlx-postgres",            # `DATABASE_DRIVER` feature
 ] }
+
+moka = { version = "0.11.2", features = ["future"] }
```

Comment:
As you mentioned offline, moka is a big dependency and it sounds like an overkill for our simple use-case. That said, I don't mind leaving it to get a feel and learn. We eventually should standardise all internal caches on a single library whenever possible.